contracts/event/LemonadeEventAttestation.sol

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@ethereum-attestation-service/eas-contracts/contracts/IEAS.sol";
+import "@ethereum-attestation-service/eas-contracts/contracts/resolver/SchemaResolver.sol";
+import "@ethereum-attestation-service/eas-contracts/contracts/resolver/examples/AttesterResolver.sol";
+
+import "./ILemonadeEventAttestation.sol";
+import "./resolvers/EventHostSchemaResolver.sol";
+import "./resolvers/TicketIssuingSchemaResolver.sol";
+
+contract Event {}
+
+contract LemonadeEventAttestation is
+    OwnableUpgradeable,
+    ILemonadeEventAttestation
+{
+    using ECDSA for bytes;
+    using ECDSA for bytes32;
+
+    bytes32 public eventCreatorSchemaId;
+    bytes32 public eventDetailSchemaId;
+    bytes32 public eventCohostSchemaId;
+    bytes32 public ticketTypeSchemaId;
+    bytes32 public ticketSchemaId;
+
+    IEAS internal eas;
+
+    event EventCreated(
```

Comment:
You might wanna use indexed events, which allows you to find events by a specific filter. So if you want to be able to find events by creator then you have to do `address indexed creator`.

Reply:
actually we won't need this find action, the search actions are supported by the EAS graphql server so we're good